.github/workflows/release-github.yaml

```diff
@@ -0,0 +1,92 @@
+name: Build and upload python packages to Github release
+
+on:
+  workflow_dispatch
+
+jobs:
+  build-wheels-macos:
+    runs-on: macos-10.15
+    env:
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: "true"
+      MINICONDA_FILENAME: Miniconda3-latest-MacOSX-x86_64.sh
+      CC: clang
+      CXX: clang++
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Download and install miniconda
+      shell: bash
+      run: |
+        curl -L -o $MINICONDA_FILENAME "https://repo.continuum.io/miniconda/$MINICONDA_FILENAME"
+        bash ${MINICONDA_FILENAME} -b -f -p $HOME/miniconda3
+        echo "::add-path::$HOME/miniconda3/bin"
+        echo "::add-path::$HOME/miniconda3/Scripts"
+        conda init
+
+    - name: Download and build dependencies
+      run: |
+        eval "$(conda shell.bash hook)"
+        conda create -n meson python=3.8
+        conda activate meson
+        conda install -y meson
+        make download-build-static-deps
+        conda deactivate
+
+    - name: Build and upload packages
+      run: |
+        eval "$(conda shell.bash hook)"
+        for VERSION in 3.6 3.7 3.8 3.9 3.10; do
+          conda create -n py$VERSION python=$VERSION
+          conda activate py$VERSION
+          python setup.py build_ext bdist_wheel
+          conda deactivate
+        done
+
+    - uses: actions/upload-artifact@v3
+      with:
+        name: macos-wheels
+        path: dist/*.whl
+
+
+  build-wheels-linux:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Build package and upload from docker
+      run: |
+        docker run --rm -v "${PWD}:/opt/fast-mosestokenizer" \
+          -e TWINE_USERNAME=__token__ \
+          -e TWINE_PASSWORD=${{ secrets.PYPI_TOKEN }} \
```

Comment:
I don't think these lines are really necessary. That script doesn't upload anything anymore.